access/src/main/external/static/recordTransformations/modsToFullRecord.xsl

```diff
@@ -544,15 +549,15 @@
 		</xsl:for-each-group>
 	</xsl:template>
 
-	<!-- mods:location -->
+	<!-- mods:location  p.s. why is there uri here?  that isn't a valid mods element. sn March 2015 -->
 	<xsl:template name="modsLocations">
 		<xsl:for-each-group select="*[local-name() = 'location']" group-by="@displayLabel, local-name(.[not(@displayLabel)])[. != '']">
 			<xsl:variable name="groupKey" select="current-grouping-key()"/>
 			<tr>
 				<th>
 					<xsl:choose>
-						<xsl:when test="$groupKey = name()">
-							<xsl:text>Location</xsl:text>
+						<xsl:when test="$groupKey = local-name()">
+							<xsl:text>File location</xsl:text>
```

Comment:
Just a question, is this always going to be the location of a file?  I don't know the correct usage of this element